### ISSUE TITLE
Promotion

### ISSUE THREAD

**Issue by dkeeney** [htm-community/nupic.cpp#389]
@ctrl-z-9000-times and @breznak 
Matt Taylor is giving a love streaming session on Fri 12th at 9am Pacific...I think on Twitch.  
You should have received an email with the link.  I will be traveling so I cannot attend.  Can either or both of you attend to promote the repository?

I am traveling to San Jose CA to see my Son and his family, about a 4 hr drive.  I will be leaving tomorrow (Wed, 10th) and will not return until Sunday.  I will not have access to my computer during that time so will not get much done.

The following week (April 22 - 24) we are taking a run down to Algodones Mexico for some dental work.  Then April 27 - 30) we will be in San Jose CA again.  So this will be a busy month.

**Comment by breznak** [comment]
Thanks for the heads up!, I did not get an email, or missed it. 
I'll try to attend, although not sure I can be avail., at least I know about it now. 

Enjoy your travels (and the dentist, if possible) :), I envy you the sunny California now :sunny:

**Comment by ctrl-z-9000-times** [comment]
Thanks for attending breznak!

**Comment by dkeeney** [comment]
Yes, thanks for attending.  It looks like our work may pay off. 👍 
The takeaways for me were:
- I am glad you picked up on the fact that dynamic libraries were a problem.
- we should be looking for any changes might be needed to work with htmresearch.
- We also need to consider changing from nupic namespace to htm namespace as Matt suggested.
- Need to publish to pypi using a wheel.
- We need to flush out our wiki to describe how to do things using our library.

**Comment by breznak** [comment]
> we should be looking for any changes might be needed to work with htmresearch. 

yes, this is a priority for me too. On the other hand, does not depend on us, if they want to use .core, or our .cpp. We can talk to Matt what's the mood and how are the chances. 

> We also need to consider changing from nupic namespace to htm namespace as Matt suggested. 

missed that, why?

**Comment by dkeeney** [comment]
Matt wants to keep the `nupic` namespace for htmresearch and be separate from us.  He may have been thinking PYPI namespace but I think it should also apply to the C++ namespace.  The purpose of a namespace is to avoid collision of names.  If they want to control the `nupic` namespce then we need something else. `htm` seems like a likely candidate if we can get it from PYPI.
